nix/checks.nix

```diff
@@ -51,6 +51,7 @@ inputs: pkgs: {
     buildPhase = "make check-format";
     installPhase = "touch $out";
     meta.checkDescription = "that OCaml code is formatted properly";
+    DISABLE_CHECK_OPAM_SWITCH = "true";
```

Comment:
I'm not entirely convinced that we should skip the check here: using the wrong `opam.export` may give us different formatting, depending on the version of ocamlformat.

Reply:
For this check, the environment is managed via nix and not opam, so the `check_opam_switch` tool does not work in this case.

It should not be needed however as nix directly reads the `opam.export` file to setup the environment.